VMBackup/main/patch/__init__.py

```diff
@@ -19,6 +19,7 @@
 import os
 import re
 import platform
+import traceback
```

Comment:
this import is unwanted

Reply:
the lib is used line 64, it's something i came across during debugging